bin/BuildPackages.sh

```diff
@@ -8,26 +8,45 @@ mkdir -p "$LOGDIR"
 LOGFILE=buildpackages
 FAILPKGFILE=fail
 
-# print notices in green
-notice() {
-    printf "\033[32m%s\033[0m\n" "$@"
-}
-
-# print warnings in yellow
-warning() {
-    printf "\033[33mWARNING: %s\033[0m\n" "$@"
-}
-
-# print error in red and exit
-error() {
-    printf "\033[31mERROR: %s\033[0m\n" "$@"
-    exit 1
-}
-
-# print stderr error in red but do not exit
-std_error() {
-    printf "\033[31mERROR: %s\033[0m\n" "$@"
-}
+if [ -t 1 ]; then
```

Comment:
Should not this ````if```` be within each command ````notice````, ````warning````, ````error````? I have the feeling if run as it is, then it checks at line 11 if terminal is open, and if yes, then the ````notice```` will be a command that always colours, and if not, then ````notice```` never colours.